simphony_mayavi/core/tests/test_cell_collection.py

```diff
@@ -0,0 +1,145 @@
+import unittest
+
+from tvtk.api import tvtk
+
+from simphony_mayavi.core.cell_collection import CellCollection
+
+
+class TestCellCollection(unittest.TestCase):
+
+    def setUp(self):
+        self.cells = [
+            [0, 1, 2, 3],
+            [4, 5, 6, 7, 8, 9, 10, 11],
+            [2, 7, 11],
+            [1, 4],
+            [1, 5, 8]]
+
+        self.vtk = tvtk.CellArray()
+        self.vtk.from_array(self.cells)
+
+    def test_length(self):
+        # when
+        collection = CellCollection()
+
+        # then
+        self.assertEqual(len(collection), 0)
+
+        # when
+        array = tvtk.CellArray()
+        array.insert_next_cell(tvtk.Line())
+        array.insert_next_cell(tvtk.Line())
+        collection = CellCollection(cell_array=array)
+
+        # then
+        self.assertEqual(len(collection), 2)
+
+    def test_getitem(self):
+        # given
+        cells = self.cells
+        collection = CellCollection(self.vtk)
+
+        # when/then
+        for index in range(5):
+            self.assertSequenceEqual(collection[index], cells[index])
+
+        # when/then
+        with self.assertRaises(IndexError):
+            collection[5]
+
+    def test_getitem_with_long_int_index(self):
+        # given
+        cells = self.cells
+        collection = CellCollection(self.vtk)
+
+        # when/then
```

Comment:
This one is correct, I have updated the other places were when/then was used by mistake